app/views/moderations/mod.html.erb

```diff
@@ -54,6 +54,20 @@
       This is the equivalent of vomiting on <b>all</b> of this user's articles.
     </p>
   </div>
+  <% if @hidden_comments.present? %>
+    <div class="tag-mod-form">
+      <h2>Hidden Comments</h2>
+      <form>
```

Comment:
Don't hate me for abusing a `form` tag, I didn't feel like adding more CSS, especially since this feature may still evolve.